src/styling/fonts.ts

```diff
@@ -1,4 +1,6 @@
-export const setupFonts = `
+/* eslint-disable global-require */
```

Comment:
It was my bad. Is it possible to use imports instead of requires here?

Reply:
I was trying to do this but I didn't find the way. So I just disabled the rule.

Reply:
Ok. But I think we should get back to this later.